{"name": "Login con credenciales incorrectas -- @1.1 ", "status": "broken", "statusDetails": {"message": "\nYou can implement step definitions for undefined steps with these snippets:\n\n@then(u'aparece \"error\" en pantalla')\ndef step_impl(context):\n    raise NotImplementedError(u'STEP: Then aparece \"error\" en pantalla')\n\n"}, "steps": [{"name": "Given se ha inicializado el contexto", "status": "passed", "start": 1643965941681, "stop": 1643965943729}, {"name": "When se lanza el login con \"badUser\" y \"secret_sauce\"", "status": "passed", "start": 1643965943730, "stop": 1643965943950}, {"name": "Then aparece \"error\" en pantalla", "status": "broken", "statusDetails": {"message": "\nYou can implement step definitions for undefined steps with these snippets:\n\n@then(u'aparece \"error\" en pantalla')\ndef step_impl(context):\n    raise NotImplementedError(u'STEP: Then aparece \"error\" en pantalla')\n\n"}, "attachments": [{"name": ".text", "source": "05f036e4-ccd2-49e6-bf50-a60175989c89-attachment.txt", "type": "text/plain"}], "start": 1643965943950, "stop": 1643965943951}], "parameters": [{"name": "usuario", "value": "badUser"}, {"name": "contrasena", "value": "secret_sauce"}, {"name": "elemento", "value": "error"}], "start": 1643965941680, "stop": 1643965944005, "uuid": "78a8e24d-6912-4861-95f5-10a1e18a5312", "historyId": "927c8a39cb7604db2d853930dde257ab", "fullName": "Pruebas sobre el login: Login con credenciales incorrectas", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Pruebas sobre el login"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}